Library/Homebrew/dependency_collector.rb

```diff
@@ -18,7 +18,7 @@
 class DependencyCollector
   # Define the languages that we can handle as external dependencies.
   LANGUAGE_MODULES = Set[
-    :chicken, :jruby, :lua, :node, :ocaml, :perl, :python, :python3, :rbx, :ruby
+    :lua, :perl, :python, :python3, :ruby
```

Comment:
Thanks Misty. I'll try and stick it back on my pile at some point soon so it doesn't sit there indefinitely without longer-term justification.